types/splitio.d.ts

```diff
@@ -346,7 +346,7 @@ declare namespace SplitIO {
    * @see {@link https://help.split.io/hc/en-us/articles/360058730852-Browser-SDK#attribute-syntax}
```

Comment:
`@typedef {Object.<number, string, boolean, string[], number[]>} Attributes` --> `@typedef {Object.<AttributeType>} Attributes`

Reply:
Let's change it in the release branch